test/native/integration/editor-history.native.js

```diff
@@ -42,17 +62,17 @@ describe( 'Editor History', () => {
 	` );
 
 		// Act
-		fireEvent.press( screen.getByLabelText( 'Undo' ) );
-		fireEvent.press( screen.getByLabelText( 'Undo' ) );
-		fireEvent.press( screen.getByLabelText( 'Undo' ) );
+		toggleUndo();
```

Comment:
👋🏻 @geriux. Providing a friendly note — merely for awareness — that these changes introduced several `act` warnings to the `editor-history` tests, presumably because the invocation leads to state updates that re-render the editor tree. I addressed the new warnings in https://github.com/WordPress/gutenberg/pull/52446/commits/ffb9837ed532640126b10c85d7ed97d889826031.

Reply:
Oh thanks for taking care of that!